web-client.html

```diff
@@ -34,9 +39,15 @@
 
 </li>
 
-<li><p class="first">To view an image in the <span class="bold">Image Viewer</span>, select the image in the data tree and click on the <span class="bold">Launch Full Viewer</span> button.</p>
+<li><p class="first">To view an image in the
+    <span class="bold">Image Viewer</span>, select the image in the data tree
+    and click on the <span class="bold">Launch Full Viewer</span> button.</p>
 
-<p>or double click on the image or thumbnail.</p>
+<p>Alternatively, double click on the image or thumbnail, or right-click and
+    use the <span class="bold">Open With...</span> menu to select the
+    OMERO.web app to use (this feature is new in OMERO 5.3.0 and any
+    additional apps such as OMERO.figure will show up here once your system
```

Comment:
only if they configure it to be available from open with
i.e.
``
bin/omero config append omero.web.open_with '["omero_figure", "new_figure",
      {"supported_objects":["images"], "target": "_blank", "label": "OMERO.figure"}]'
``